### DISCUSSION TITLE
Unbundle Studio from CLI?

### DISCUSSION THREAD

**Discussion by jstcki** [chakra-ui/panda#1693] [Ideas]
Currently, when installing the CLI `@pandacss/dev`, Panda Studio is also installed, which itself has its own set of dependencies (most notably [Astro](https://astro.build/)). This is a bit unexpected – I only noticed the Astro install by accident and wondered where it came from.

According to yarn, the Studio dependency and its dependencies take up 387MB of space, most of which is due to the Astro install. That seems a bit much for a tool that's purely optional.

So, to save space and bandwidth, I think it would be preferable to remove Studio as a hard dependency of the CLI. People who want to use it could easily add it explicitly. What do you think?

**Comment by segunadebayo** [comment]
This is now done

**Reply by jstcki** [reply]
Awesome, thanks!
